### DISCUSSION TITLE
[feature request] New automerge type: `pr-label`

### DISCUSSION THREAD

**Discussion by kkom** [renovatebot/renovate#24656] [Suggest an Idea]
Graphite merge queue, unlike `bors-ng`, uses PR labels instead of comments to add a PR to the queue:

https://graphite.dev/docs/graphite-merge-queue

It would be amazing to have a new [`automergeType`](https://docs.renovatebot.com/configuration-options/#automergetype) option of `pr-label` to integrate with it. What do you think?

**Comment by rarkins** [comment]
Created feature request #24663. PRs welcome
